### ISSUE TITLE
collection.url in registrationCollectionMixin should use _.result

### ISSUE THREAD

**Issue by kentmw** [vecnatechnologies/backbone-torso#110]
```
          return $.ajax({
              type: collection.fetchHttpAction,
              url: collection.url + collection.getByIdsUrl,
              contentType: 'application/json; charset=utf-8',
              data: JSON.stringify(idsToFetch)
            })
```
uses collection.url straight and doesn't allow for a method

**Comment by kentmw** [comment]
+1 had this happen to me again... My function returning a url failed.

**Comment by mandragorn** [comment]
Fix in both 0.1.x and master branches.

**Comment by kentmw** [comment]
https://github.com/vecnatechnologies/backbone-torso/pull/138
